src/TisaBackend/TisaBackend.Domain/Models/Airline.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace TisaBackend.Domain.Models
+{
+    public class Airline
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+
+        public IList<Airplane> Airplanes { get; set; }
```

Comment:
Why do we want to keep both the list of planes here and also to hold for each plane to which company it belongs?

Reply:
It is one to many and many to one relationship in efcore (represents the database)